libtests/qtest/json_parse.test

```diff
@@ -120,15 +120,15 @@ my @bad = (
     "stray low surrogate",      # 37
     "high high surrogate",      # 38
     "dangling high surrogate",  # 39
-    "duplicate dictionary key", # 40
+    "bs char in string",        # 40
     "decimal point after minus",# 41
     "e after minus",            # 42
     "missing digit after e",    # 43
     "missing digit after e+/-", # 44
     "tab char in string",       # 45
     "cr char in string",        # 46
     "lf char in string",        # 47
-    "bs char in string",        # 48
+    #"bs char in string",       # 40
```

Comment:
I think you forgot to delete this comment.

Reply:
I left it deliberately as a comment because logically the test belongs there.

Reply:
Please do this instead. It's ugly that I set it up with the tests being in a specific order, but I don't want random renames or comments that would be in the way if we added more tests.

https://github.com/m-holger/qpdf/pull/7